docs/graphene-workload-tutorial/README.md

```diff
@@ -0,0 +1,64 @@
+<!--
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+-->
+
+# Writing a Python workload to run with Graphene SGX
+
+## Creating the workload from template
+
+For this tutorial let us consider a check for palindrome as the business logic for our workload.
+Bring up the docker container from current directory with the name of your workload as an environment variable. For example -
+```
+WORKLOAD_NAME=palindrome docker-compose up
+```
+This will prepare your the setup for building your workload. Note that if you want to create another workload (e.g. - smallbank), you just need to replace the business logic in `smallbank/src/smallbank.py`. You would find a function named `_smallbank()` here which conatains the core logic.
```

Comment:
conatains -> contains

Reply:
Not sure why we want to introduce smallbank here. I though palindrome is an example to show how to add a new graphene python workload.

Reply:
No more there.